src/legacy/components/LoadingScreen.tsx

```diff
@@ -500,7 +500,6 @@ class LoadingScreen extends Component<any, any> {
                   </div>
                 </div>
 
-                {/*
```

Comment:
This was a bugfix we introduced a couple of days ago, please do not override it. I would recommend merging origin master into your branch